astro/src/content/docs/apis/jwt.mdx

```diff
@@ -239,6 +239,11 @@ The refresh token may be provided either in the HTTP request body or as a cookie
   <APIField name="token" type="String" optional>
     The previously issued encoded access token.  When provided on the request, this value will be relayed in the related JWT Refresh webhook event within the `original` field.
   </APIField>
+  <APIField name="timeToLiveInSeconds" type="Integer" optional since="1.99.9">
+    The time to live for the new access token.
+
+    When provided on the request, this value will override the default <InlineField>tenant.jwtConfiguration.timeToLiveInSeconds</InlineField> or <InlineField>application.jwtConfiguration.timeToLiveInSeconds</InlineField> values, but it must be less than or equal to those values.
```

Comment:
Trying to get around the incorrectishness of "but it must be less than or equal to those values", when it only needs to be less than the one that is being enforced. What happens if you request an invalid TTL? Should we not that here, too?

```suggestion
    When provided on the request, this value will override the default TTL, provided it is less than the default value. The default TTL is taken from <InlineField>application.jwtConfiguration.timeToLiveInSeconds</InlineField> if it is set there, otherwise it is taken from  <InlineField>tenant.jwtConfiguration.timeToLiveInSeconds</InlineField>.
```

Reply:
04faab7a927372bda600ea31ae81f037bb0c61b5 - tried to capture invalid there. although neither backticks or code worked to describe tenant or app.